std/algebra/twistededwards/bandersnatch/point.go

```diff
@@ -27,9 +29,16 @@ type Point struct {
 	X, Y frontend.Variable
 }
 
+// Set a point to a point
+func (p *Point) Set(api frontend.API, p1 *Point) *Point {
+	p.X = p1.X
+	p.Y = p1.Y
+	return p
+}
+
 // Neg computes the negative of a point in SNARK coordinates
 func (p *Point) Neg(api frontend.API, p1 *Point) *Point {
-	p.X = api.Neg(p1.X)
+	p.X = api.Sub(0, p1.X)
```

Comment:
is this actually any different than previous `Neg` call?

Reply:
No there is no difference. I changed this while debugging and forgot to change it back.